scrapy_zyte_api/_middlewares.py

```diff
@@ -31,7 +31,7 @@ def slot_request(self, request, spider, force=False):
             return
 
         downloader = self._crawler.engine.downloader
-        slot_id = downloader._get_slot_key(request, spider)
+        slot_id = downloader.get_slot_key(request)
```

Comment:
We need to keep support for older versions of Scrapy as well:

```suggestion
        try:
            slot_id = downloader.get_slot_key(request)
        except AttributeError:
            slot_id = downloader._get_slot_key(request, spider)
```